source/android/adaptivecards/src/main/res/layout/adaptive_card_select_item.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@android:id/text1"
```

Comment:
is it possible to give a better name to this string, is it a Title or something specific?

Reply:
updated